### ISSUE TITLE
[Help] ZHA read cluster attributes

### ISSUE THREAD

**Issue by zandadoum** [zigpy/zha-device-handlers#585]
Hello,

I have a plug that is not totally supported yet (https://github.com/zigpy/zha-device-handlers/issues/516)

but while that gets fixed or not, some of the missing info is actually visible manually by going into ZHA->Device->manage clusters.

for example on my plug, if i select "analog input endpoint id 21) on the dropdown, then "present_value (id: 0x0055)" and then click on the "get zigbee attribute" button, it actually gives me the value that i need...

but... how can i transform this into a sensor, or read this attribute in an automation/template/script or whatever?

basically, what i need is a way to do what the "get zigbee attribute" button does behind the scenes.

could anyone help?
